tt_metal/hostdevcommon/common_runtime_address_map.h

```diff
@@ -17,9 +17,6 @@
 constexpr static std::uint32_t L1_KERNEL_CONFIG_BASE = MEM_MAP_END;
 constexpr static std::uint32_t L1_KERNEL_CONFIG_SIZE = 69 * 1024;
 
-constexpr static std::uint32_t NUM_CIRCULAR_BUFFERS = 32;
-constexpr static std::uint32_t UINT32_WORDS_PER_CIRCULAR_BUFFER_CONFIG = 4;
-
 // Helper functions to convert NoC coordinates to NoC-0 coordinates, used in metal as "physical" coordinates.
 #define NOC_0_X(noc_index, noc_size_x, x) (noc_index == 0 ? (x) : (noc_size_x-1-(x)))
```

Comment:
this can be addressed in a separate PR, but if this got moved behind the HAL in this PR then common_runtime_address_map wouldn't have to be added as an include elsewhere

Reply:
The primary purpose of this PR was to move these two defines to `circular_buffer.h`. So that firmware header files are the owner of this constant.

Reply:
@pgkeller I have the follow up PR ready:

https://github.com/tenstorrent/tt-metal/pull/14920